business/aggregator.js

```diff
@@ -66,6 +69,22 @@ class AggregationService {
     const latest = getLatestVersion(versions)
     return latest ? { toolSpec: `${tool}/${latest}`, summary: summarized[tool][latest] } : null
   }
+
+  /*
+   * Take the clearlydefined tool as the source of truth for file paths as it is just a recursive dir
+   * Intersect the summarized file list with the clearlydefined file list by path
+   */
+  _normalizeFiles(result, summarized, coordinates) {
+    const cdFiles = get(this._findData('clearlydefined', summarized), 'summary.files')
+    if (!cdFiles) return
+    const difference = differenceBy(result.files, cdFiles, 'path')
+    if (!difference.length) return
+    this.logger.info('difference between summary file count and cd file count', {
+      count: difference.length,
+      coordinates: coordinates.toString()
+    })
+    result.files = intersectionBy(result.files, cdFiles, 'path')
```

Comment:
order sometimes matters in these operations. have you looked at that?

Also, can we get the difference length by taking `result.files.length - intersection.length`? At least in theory that should be the same. Would just save us some time on larger file lists.